Vagrantfile

```diff
@@ -57,15 +57,15 @@ Vagrant.configure("2") do |config|
     sudo apt-get update
     sudo apt-get install -y code
     code --install-extension hackwaly.ocaml
-    # fix launching from GUI
+    # fix launching from GUI (use only one forward-slash if executing manually)
```

Comment:
Maybe it would safer if we quited it like `sed -i -e '...'`, unless these instructions are copied from somewhere, and we want to preserve them as they are.